### PR TITLE
Adding Workflow Designer files to .NET Framework 4.7.2 sources

### PR DESCRIPTION
**Pull request by rbhanda**
# READ BEFORE FILING

This repository is read-only and doesn't accept community pull requests.

### CODE REVIEW

File: Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs [hunk 1]
```diff
@@ -0,0 +1,242 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
```
Comment (jimcarley) [inline Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs]: You seem to be missing the files directly under Microsoft.Activities.Build:

Microsoft.Activities.Build.csproj
Microsoft.Activities.Build.targets
Settings.StyleCop
SR.resx

File: System.Activities.Core.Presentation/System/Activities/Core/Presentation/SwitchDesigner.xaml [hunk 2]
```diff
@@ -0,0 +1,195 @@
+﻿<? Copyright (c) Microsoft Corporation.  All rights reserved.?>
+<sad:ActivityDesigner x:Uid="sad:ActivityDesigner_1" x:Class="System.Activities.Core.Presentation.SwitchDesigner"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```
Comment (jimcarley) [inline System.Activities.Core.Presentation/System/Activities/Core/Presentation/SwitchDesigner.xaml]: You seem to be missing the following from System.Activities.Core.Presentation\System\Activities\Core\Presentation

TransactionScopeDesigner.xaml
TransitionDesigner.xaml
TryCatchDesigner.xaml
VerticalConnector.xaml
WhileDesigner.xaml
WriteLineDesigner.xaml

File: Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs (hunk 1 above)
Comment (jimcarley) [reply Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs]: Oops. Microsoft.Activiites.Build.Targets should be Microsoft.WorkflowBuildExtensions.targets

Comment (rbhanda) [reply Microsoft.Activities.Build/Microsoft/Activities/Build/BeforeInitializeComponentExtension.cs]: They are now added to the PR

File: System.Activities.Presentation/InternalApis/Clr/inc/AppContextDefaultValues.cs [hunk 3]
```diff
@@ -0,0 +1,180 @@
+﻿// ==++==
+//
```
Comment (jimcarley) [inline System.Activities.Presentation/InternalApis/Clr/inc/AppContextDefaultValues.cs]: I am not finding this file in either the 4.7.2 or 4.8 TF branches. Where did this come from?

File: System.Activities.Presentation/InternalApis/Clr/inc/LocalAppContext.cs [hunk 4]
```diff
@@ -0,0 +1,138 @@
+﻿// ==++==
```
Comment (jimcarley) [inline System.Activities.Presentation/InternalApis/Clr/inc/LocalAppContext.cs]: This file isn't in the branches, either.

**Pull request merged**